spec/index.html

```diff
@@ -323,9 +323,19 @@ <h2>A Canonical form of N-Quads</h2>
       and <code>graphLabel</code>,
       any of which MUST be a single space (<code>U+0020</code>).</li>
     <li><a data-cite="RDF12-CONCEPTS#dfn-literal">Literals</a> with the
-      datatype <code>http://www.w3.org/2001/XMLSchema#string</code>
+      datatypes <code>http://www.w3.org/2001/XMLSchema#string</code> and
+      <code>http://www.w3.org/1999/02/22-rdf-syntax-ns#langString</code>
       MUST NOT use the datatype IRI part of the <a href="#grammar-production-literal">literal</a>,
-      and are represented using only <a href="#grammar-production-STRING_LITERAL_QUOTE">STRING_LITERAL_QUOTE</a>.
+      and are represented using only <a href="#grammar-production-STRING_LITERAL_QUOTE">STRING_LITERAL_QUOTE</a>
+      and <a href="#grammar-production-LANGTAG">LANGTAG</a>,
+      as appropriate.
+      <div class="issue" data-number="45">
+        [[RDF12-CONCEPTS]] allows implementations to convert
+        <a data-cite="RDF12-CONCEPTS#dfn-language-tag">language tags</a> to lower case.
+        This represents a potential interoperability issue if language tags
+        are not converted to lower case uniformly, as different implementations
+        may end up using different canonical representations of such language tags.
+      </div>
```

Comment:
This is note is not accurate.

By changing the **output**, the N-quads file will not necessarily be the same graph as the same data, written as TriG or non-canonical, n-quads even if read back into the same producing system.

The canonical representations of language tags as defined by RFC 5646 involves other steps.